Kernel/API/Syscall.h

```diff
@@ -474,6 +476,17 @@ struct SC_chmod_params {
     int follow_symlinks;
 };
 
+enum class scheduler_parameters_mode : bool {
```

Comment:
Enums names are usually PascalCase, aren't they?

Reply:
I was unsure because of the syscall context, but if you say so sure